### PR TITLE
Use tt font in example messages

### PR DESCRIPTION
**Pull request by rocky**
Go over history and what is missing pages

<a href="https://gitpod.io/#https://github.com/Mathics3/mathics-core/pull/476"></a>

### CODE REVIEW

**Comment by rocky** [comment]
There is some breakage in the Tracing example, but I'll just have to leave that for some other time. So much to do.

File: mathics/doc/documentation/1-Manual.mdoc [hunk 1]
```diff
@@ -49,28 +50,32 @@ Some of the features of \Mathics are:
 
 </section>
 
-<section title="What is missing?">
-There are lots of ways in which \Mathics could still be improved.
-
-Most notably, performance is still slow. Although there are various ways to speed up Python, some serious work is need in \Mathics, to speed it up. This will be addressed in the future.
+<section title="History">
+The version alpha version of \Mathics was done in 2011 by Jan Pöschko. He worked on it for a couple of years to about v0.5 which had 386 built-in symbols.
```
Comment (mmatera) [inline mathics/doc/documentation/1-Manual.mdoc:54]: word repeated.

File: mathics/doc/documentation/1-Manual.mdoc [hunk 2]
```diff
@@ -49,28 +50,32 @@ Some of the features of \Mathics are:
...
-\skip
-Welcome to \Mathics, have fun!
-</section>
+<ul>
+<li>Ensure this document is complete and accurate. We could use help to ensure all of the Builtin functions described properly and fully, and that they have link to corresponding Wiki, Sympy, WMA and/or mpath links.
+Are the builtin summaries and examples clear and useful</li>
+<li>We could use help in LaTeX styling, and going over this document to remove overful boxes and things of that nature. We could also use help and our use of Asymptote. The are some graphics primitives such as for polyhedra that haven't been implemented. Similar graphics options are sometimes missing in Aymptote that we have available in other graphics backends.</li>
 
-</chapter>
+<li>write a grpahics backend favorite graphics backend: it could be a javascript library like jsiddle</li>
```
Comment (mmatera) [inline mathics/doc/documentation/1-Manual.mdoc:74]: Typo

Comment (rocky) [reply mathics/doc/documentation/1-Manual.mdoc:74]: Thanks both should be corrected in 09596a10

**Pull request merged**